src/main/java/org/jenkinsci/plugins/configfiles/maven/security/CredentialsHelper.java

```diff
@@ -73,6 +78,7 @@ private CredentialsHelper() {
     }
 
     public static List<StandardUsernameCredentials> findValidCredentials(final String serverIdPattern) {
+        // TODO Cyrille Le Clerc 2016/05/08: why do we return an empty list of server id is empty?
```

Comment:
I guess we could also just return all - true. Will you adjust the PR to reflect it?